.github/workflows/docker-build-static.yml

```
@@ -425,6 +443,33 @@ jobs:
     needs: [build-ui, build-bin]
     name: Build Docker image
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        container: ["debian","alpine"]
+        include:
+          - container: alpine
+            platforms: linux/amd64,linux/arm64
+            suffix: "-alpine"
+            tags: |
+                  type=ref,event=pr
+                  type=semver,pattern=v{{version}}
+                  type=semver,pattern=v{{major}}
+                  type=semver,pattern=v{{major}}.{{minor}}
```

```suggestion
                  type=semver,pattern=v{{version}}
                  type=semver,pattern=v{{major}}
                  type=semver,pattern=v{{major}}.{{minor}}
                  type=semver,pattern=v{{version}},suffix=
                  type=semver,pattern=v{{major}},suffix=
                  type=semver,pattern=v{{major}}.{{minor}},suffix=
```

I believe we still want version tags without base OS too.

Very nice idea with moving the tags to the matrix! An further change you COULD do (feel free to ignore if yo uwant):

Remove the suffix from the matrix, and just hardcode the suffix into the list of tags here and below. Then you can remove the suffix from the flavor field in the metadata step. My thinking here is the matrix is providing the list of tags anyways - why continue to split that logic between the matrix definition and the metadata step